commonjs/frontend-core.js

```diff
@@ -1,60 +1,3 @@
-//source: https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
-if (!Array.prototype.forEach) {
```

Comment:
why? shim is only for ie8, in ie9 it only works without strict mode? pls comment it anywhere